src/mono/wasm/runtime/marshal.ts

```diff
@@ -334,7 +334,7 @@ export class ManagedError extends Error implements IDisposable {
 
     getManageStack() {
         const gc_handle = (<any>this)[js_owned_gc_handle_symbol];
-        if (gc_handle) {
+        if (gc_handle && runtimeHelpers.jsSynchronizationContextInstalled) {
```

Comment:
Is this for thread safety?

Reply:
The other PR will dispose the object before this scenario could happen. The handle would be zero. This is just stop gap to make the tests pass even with broken disposal. This will just not have managed stack trace after mono exit.